models/image.py

```diff
@@ -635,6 +642,69 @@ def from_exposure(cls, exposure, section_id):
 
         return new
 
+    @classmethod
+    def copy_image(cls, image):
+        """Make a new Image object with the same data as an existing Image object.
+
+        This new object does not have a provenance or any relationships to other objects.
+        It should be used only as a working copy, not to be saved back into the database.
+        The filepath should also be set to a new (unique) value so as not to overwrite the original.
```

Comment:
If I read the function correctly, the filepath will be empty.  Perhaps document that in the docstring.